README.org

```diff
@@ -136,8 +136,10 @@ This generates the following files:
  - [[file:precomputed/viridiplantae_all_2014.utax.fa][viridiplantae_all_2014.utax.fa]]
  - [[file:precomputed/viridiplantae_all_2014.utax.tax][viridiplantae_all_2014.utax.tax]]
 The first three are also included in the precomputed folder. And the last two are included in the training/utax folder.
+The required file format changed in the new versions of usearch. The compatible file is included in 
+ - [[file:precomputed/viridiplantae_all_2014.utax.fa][viridiplantae_all_2014.sintax.fa]]
```

Comment:
The link has the correct text but actually links to the utax file not the sintax file.